src/jobs/actions/emailaction.ts

```diff
@@ -38,31 +55,49 @@ export class EmailJobAction<T> implements JobAction<T> {
       "EmailJobAction",
     );
 
-    if (!data["mailer"]) {
-      throw new NotFoundException("Param 'mailer' is undefined");
-    }
     if (!data["to"]) {
       throw new NotFoundException("Param 'to' is undefined");
     }
     if (!data["from"]) {
       throw new NotFoundException("Param 'from' is undefined");
     }
     if (typeof data["from"] !== "string") {
-      throw new TypeError("from should be a string");
+      throw new TypeError("Param 'from' should be a string");
+    }
+    if (!data["password"]) {
```

Comment:
Password should be optional, as not all mailhosts do require one.  And From and auth may differ.

I would suggest an optional auth field that can be passed as-is into the  auth member of createTransport.
This also allows to configure the other possible auth schemes easier.

Reply:
Thanks for the suggestion @bpedersen2, I added the optional `auth` field in the latest commit https://github.com/SciCatProject/scicat-backend-next/pull/1326/commits/64c897195a5d3cd1b967dca4bdcc9a94870d3c77